README.md

```diff
@@ -1,56 +1,53 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# SOLOVINO TV SHOWS
 
-> Description the project.
+> This project is a place where you can like and comment your favorites TV Shows.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- JavaScript, CSS, HTML.
+- Jest, Babel.
+- Visual Studio Code.
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://diegomero.github.io/javascript-capstone/dist/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+1. Open Git and go to the folder you want to clone this repo.
 
-### Setup
-
-### Install
+2. Type git clone git@github.com:DiegoMero/javascript-capstone.git
 
-### Usage
+3. Now you can use this repo.
 
-### Run tests
+### Prerequisites
 
-### Deployment
+You will only need a working pc.
 
+### Install
 
+Feel free to fork this repo!
```

Comment:
- [ OPTIONAL ] Great job adding the `Getting Started` section 💯. However,  I suggest adding more instructions about how to run your project, so I suggest adding `npm install` to install all dependencies, `npm run start` to run the project, and  `npm run test` to run the test 👌